syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -143,7 +143,7 @@ def restart(self, time_stamp: float):
 
 
 class HyperbandScheduler(FIFOScheduler, MultiFidelitySchedulerMixin):
-    """Implements different variants of asynchronous Hyperband
+    r"""Implements different variants of asynchronous Hyperband
```

Comment:
Is the `r` intentional? There are few more in the PR as well

Reply:
Yes, this is for comments with contain `\`, otherwise we are getting warning outputs in the tests.